composer.json

```diff
@@ -26,6 +26,7 @@
         "aura/di": "^3.2",
         "composer/composer": "^1.3.2",
         "filp/whoops": "^2.1.7",
+        "kocal/zend-expressive-doctrinedatabase": "^1.0",
```

Comment:
It's all fine that you created your own doctrine factory, but why not use [DASPRiD/container-interop-doctrine](https://github.com/DASPRiD/container-interop-doctrine)? With that one you can configure all of the doctrine features.